aten/src/ATen/native/AutogradComposite.cpp

```diff
@@ -92,18 +92,12 @@ bool _has_same_storage_numel(const at::Tensor& base, const at::Tensor& other) {
 }
 
 Tensor _lazy_clone(Tensor const& self) {
-  c10::StorageImpl* self_storage = self.storage().unsafeGetStorageImpl();
-  c10::intrusive_ptr<c10::StorageImpl> storage =
-    c10::impl::cow::lazy_clone_storage(*self_storage);
-  TORCH_CHECK(storage != nullptr);
-  auto tensor = c10::make_intrusive<c10::TensorImpl>(
-      c10::Storage(std::move(storage)),
-      self.key_set(),
-      self.dtype());
-  tensor->set_sizes_and_strides(self.sym_sizes(),
-                                self.sym_strides(),
-                                self.sym_storage_offset());
-  return Tensor(std::move(tensor));
+  Tensor res = self.view_symint(self.sym_sizes());
```

Comment:
fwiw I don't expect this to help, because your lazy_clone is below autograd key, so the differentiable view meta (which is what _is_view is querying) won't get initialized anyway